manager-bundle/src/Resources/skeleton/config/config.yml

```diff
@@ -126,3 +126,8 @@ fos_http_cache:
         annotations:
             enabled: false
         max_header_value_length: 4096
+
+# TwoFactor configuration
+scheb_two_factor:
+    trusted_device:
+        lifetime: 15768000 # 15768000 seconds corresponds to six months
```

Comment:
how is this related to the PR?

Reply:
We don't renew the token anymore and @leofeyer wanted to have at least 6 months of validity.

Reply:
how about we add a `renewTrustedToken` to the `TrustedDeviceManager` which only calls the storage? Then we could renew as before, without causing duplicate entries.

Reply:
Provided that it can be implemented easily, I'd be fine with that. Otherwise it will not hurt if you have to reconfirm your trusted device every six months, either. IDK if forever trusted devices are a good idea in general?

Reply:
This does not work either. The storage gives no possibilty to get the token value (needed to compare previous entries in the database). So everytime we renew a token you would get a new entry in the trusted device list.

Reply:
As discussed in Mumble on Feburary 13th, we want to use the default lifetime of the bundle (60 days). https://github.com/scheb/two-factor-bundle/blob/a45b2f7ec61bbfc6c833d21b70111d75b2fb489d/Resources/doc/trusted_device.md

Reply:
Removed in a5e2f77b1.